lib/collection/src/operations/types.rs

```diff
@@ -328,6 +328,27 @@ pub struct ShardTransferInfo {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[anonymize(false)]
     pub comment: Option<String>,
+
+    /// Internal status of the shard transfer, do not expose.
+    #[serde(skip)]
+    #[schemars(skip)]
```

Comment:
Nits:
- reword the doc to make it clear that `comment` and `status` contain the same information.
- `#[serde(skip)]` implies `#[schemars(skip)]`, so the latter could be removed.

```suggestion
    /// Same as [`Self::comment`], but in a machine-readable form.
    /// For internal use only, do not expose.
    #[serde(skip)]
```

Reply:
Addressed in [8ad0c0f](https://github.com/qdrant/qdrant/pull/7714/commits/8ad0c0f073351ec874d04318998e990a0a04e6a0)